etc/convert-test-results.swift

```diff
@@ -85,7 +85,8 @@ let ignoreSuites = [
     "mongo-swift-driverPackageTests.xctest",
     // linux
     "All tests",
-    "debug.xctest"
+    "debug.xctest",
+    "Selected tests" // this shows us when --filter is used
```

Comment:
this accounts for the test output being slightly different when the `--filter` parameter is used and ensures we still parse the results into XML correctly